docker/ol-docker-start.sh

```diff
@@ -41,6 +43,9 @@ su solrupdater -c "python scripts/new-solr-updater.py \
 su openlibrary -c "scripts/coverstore-server $COVER_CONFIG \
     --gunicorn --workers 1 --max-requests 250 --bind :8081" &
 
+su openlibrary -c "scripts/openlibrary-server $CONFIG \
+                      install"
```

Comment:
I guess I'm not sure what infogami `install` does here. Do we go through a corresponding step in production deploys, @mekarpeles ?